# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: js/src/wasm/WasmCraneliftCompile.cpp
# Commit: 44b0c0bed8a6
# Full Hash: 44b0c0bed8a6b4debb950e7e1c69190883c65d60
# Author: Lars T Hansen <lhansen@mozilla.com>
# Date: 2021-10-18 16:09:29
# Regressor Bug: 1727084
# File Overlap Count: 1
# Description:
#   Bug 1727084 - Memory64 - Atomic-rmw baseline operations. r=yury
#   
#   Atomic ADD, SUB, AND, OR, XOR, XCHG, CMPXCHG, as well as WAIT and
#   NOTIFY for the baseline compiler.
#   
# ==============================================================================

diff -r fb02656bb788 -r 44b0c0bed8a6 js/src/wasm/WasmCraneliftCompile.cpp
--- a/js/src/wasm/WasmCraneliftCompile.cpp	Mon Oct 18 09:58:15 2021 +0000
+++ b/js/src/wasm/WasmCraneliftCompile.cpp	Mon Oct 18 09:58:15 2021 +0000
@@ -99,11 +99,11 @@
     case BD_SymbolicAddress::PostBarrier:
       return SymbolicAddress::PostBarrierFiltering;
     case BD_SymbolicAddress::WaitI32:
-      return SymbolicAddress::WaitI32;
+      return SymbolicAddress::WaitI32M32;
     case BD_SymbolicAddress::WaitI64:
-      return SymbolicAddress::WaitI64;
+      return SymbolicAddress::WaitI64M32;
     case BD_SymbolicAddress::Wake:
-      return SymbolicAddress::Wake;
+      return SymbolicAddress::WakeM32;
     case BD_SymbolicAddress::Limit:
       break;
   }